apiary.apib

```diff
@@ -3673,6 +3673,23 @@ Update a draft work order.
 
 + Response 204
 
+### workOrders.attachImage [POST /workOrders.attachImage]
+
+Attach an image to a draft work order.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + id: `4afb0a9c-91c6-49ed-a2e5-ce7c1e3a87fb` (string, required)
```

Comment:
@dietrichm wouldn't it be clearer if this would be `workorder_id`?

Reply:
Yes! Updated the proposal.